tests/functional/n/no/no_member_augassign.py

```diff
@@ -1,25 +1,30 @@
-"""Tests for no-member in relation to AugAssign operations."""
-# pylint: disable=missing-module-docstring, too-few-public-methods, missing-class-docstring, invalid-name
+"""Tests that no-member is not emitted for value-less annotations in relation
+   to AugAssign operations.
+"""
+# pylint: disable=too-few-public-methods, missing-class-docstring, invalid-name
+
 
 # Test for: https://github.com/pylint-dev/pylint/issues/4562
 class A:
     value: int
 
+
 obj_a = A()
-obj_a.value += 1  # [no-member]
+obj_a.value += 1
```

Comment:
A review comment got lost: Here we'd introduce a false negative, running the code raise an error:
```
Traceback (most recent call last):
  File "/home/pierre/pylint/doc/a.py", line 12, in <module>
    obj_a = A(1)
TypeError: A() takes no arguments
```